packages/storage-plugin/tests/storage-for-feature.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { RouterModule, Router } from '@angular/router';
+import { State, NgxsModule, Store, provideStates } from '@ngxs/store';
+import { Component, Injectable, NgModule, NgZone } from '@angular/core';
+import { freshPlatform, skipConsoleLogging } from '@ngxs/store/internals/testing';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { APP_BASE_HREF } from '@angular/common';
+
+import { NgxsStoragePluginModule, withStorageFeature } from '..';
+
+describe('forFeature', () => {
+  interface CounterStateModel {
+    count: number;
+  }
+
+  @State<CounterStateModel>({
+    name: 'counter',
+    defaults: { count: 0 }
+  })
+  @Injectable()
+  class CounterState {}
+
+  interface BlogStateModel {
+    pages: number[];
+  }
+
+  @State<BlogStateModel>({
+    name: 'blog',
+    defaults: { pages: [] }
+  })
+  @Injectable()
+  class BlogState {}
+
+  @Component({ template: 'This is blog' })
+  class BlogComponent {}
+
+  @NgModule({
+    imports: [RouterModule.forChild([{ path: '', component: BlogComponent }])],
+    declarations: [BlogComponent],
+    providers: [provideStates([BlogState], withStorageFeature([BlogState]))]
```

Comment:
I think that, in addition to this standalone API, we should have the `NgxsStoragePluginModule.forFeature([...])` API.

Reply:
We can't, because there's no `imports` feature on the `Route` level, similar to `providers`.